articles/java/spring-framework/configuration-properties-azure-event-hubs.md

```diff
@@ -169,6 +172,7 @@ ms.custom: devx-track-java, spring-cloud-azure, devx-track-extended-java
 > | spring.cloud.azure.eventhubs.processor.credential.client-secret                                                                       | Client secret to use when performing service principal authentication with Azure.                                                                                                                                                                                                                                                                                                                                                                                             |
 > | spring.cloud.azure.eventhubs.processor.credential.managed-identity-enabled                                                            | Whether to enable managed identity to authenticate with Azure. If true and the client-id is set, will use the client ID as user assigned managed identity client ID. The default value is `false`.                                                                                                                                                                                                                                                                            |
 > | spring.cloud.azure.eventhubs.processor.credential.password                                                                            | Password to use when performing username/password authentication with Azure.                                                                                                                                                                                                                                                                                                                                                                                                  |
+> | spring.cloud.azure.eventhubs.processor.credential.token-credential-bean-name                                                          | Custom Get the custom {@link com.azure.core.credential.TokenCredential} bean name, it's used for Service builder factory or passwordless authentication.                                                                                                                                                                                                                                                                                                                      |
```

Comment:
The part of `{@link com.azure.core.credential.TokenCredential}` is a Javadoc style. Since we want this javadoc to be a human readable string, so probably we should update the source code's java doc. @moarychan